cdap-ui/app/cdap/components/TimeRangePicker/TimeRangePicker.scss

```diff
@@ -44,6 +46,44 @@ $track-border-color: black;
 $track-radius: 5px;
 $contrast: 0;
 
+.expandable-time-range-picker {
+  position: absolute;
+  top: -22px;
```

Comment:
Can we set the position of this element based on margins and paddings, instead of using absolute positioning?

Reply:
no, because this element has to be removed from the document flow, otherwise the field list will get pushed down after this row